src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -250,3 +283,38 @@ def _abort_command(self) -> None:
     def _run_command(self, cmd_message, cmd_data=None) -> Union[dict, bytes, bool]:
         """Not used."""
         pass
+
+    def _get_instance(self):
+        return self
+
+
+class TesterPresentSender(threading.Thread):
+    """Self that continuously sends tester present messages via UDS"""
+
+    TESTER_PRESENT_SEND = b"\x3E\x00"
```

Comment:
@BKaDamien  would it make sense to make in configureable ?  So the user could select between with and without response

Reply:
Done, it does not really make sense to make it configurable here as we really do not need the response. The goal is just to avoid to return to default session or going to sleep